sdk/monitor/Azure.Monitor.Query/src/MetricsQueryClientBuilderExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Extensions;
+using Azure.Monitor.Query;
+
+namespace Microsoft.Extensions.Azure
+{
+    /// <summary> Extension methods to add <see cref="MetricsQueryClient"/> to client builder. </summary>
+    public static class MetricsQueryClientBuilderExtensions
+    {
+        /// <summary> Registers a <see cref="MetricsQueryClient"/> instance. </summary>
+        /// <param name="builder"> The builder to register with. </param>
+        /// <param name="endpoint"> The Data Collection Endpoint for the Data Collection Rule, for example https://dce-name.eastus-2.ingest.monitor.azure.com. </param>
```

Comment:
This seems to be the description for ingestion.

Reply:
updated